### DISCUSSION TITLE
RTABMAP with ORB-SLAM on Windows

### DISCUSSION THREAD

**Discussion by Dorothyweiking** [introlab/rtabmap#992] [How to]
How to run RTABMAP using ORB-SLAM odometery on Windows，this option is not currently available now.

**Comment by Dorothyweiking** [comment]
@matlabbe Can you give me some suggestions,thanks!

**Reply by matlabbe** [reply]
Is it possible to build ORB_SLAM2 on Windows? If yes, that would be a good start, then rtabmap would need to be [rebuilt from source](https://github.com/introlab/rtabmap/wiki/Installation#vcpkg) too.

**Reply by Dorothyweiking** [reply]
Ok, thank you, I will try it, now I use D435i, based on the f2m odometery to build maps, but the map sometimes has a large positioning drift, and some point clouds also be layered after loopback. Do you have any good suggestions on this problem? Is there a better choice of odometery, and whether the positioning of vio will be more accurate.

**Reply by matlabbe** [reply]
1. With D435i, I suggest to use IR-Depth mode if you don't use it like this already, not RGB-D mode. 

2. Try closing loop closures more often

3. Avoid looking directly to a white wall
